sqlstore/src/main/resources/db/migration/V8012__sequencer_update.sql

```diff
@@ -0,0 +1,25 @@
+DELETE FROM SequencingParameters WHERE SequencingParameters.name = 'v4 1×136';
+
+DELETE FROM Platform WHERE Platform.name IN ('Solid','LS454','IonTorrent','OxfordNanopore');
```

Comment:
Do we want to fully delete these before migration? We do have some historical Solid and IonTorrent data.